js-extensions/src/ExtensionRenderer.jsx

```diff
@@ -1,6 +1,6 @@
 var React = require('react');
 var ReactDOM = require('react-dom');
-var ExtensionStore = require('./ExtensionStore.js');
+var ExtensionStore = require('./ExtensionStore.js').instance;
```

Comment:
I'm looking at this `instance` export and I don't really see the need for it Vs just using statics (same on `ClassMetadataStore`) e.g. `ExtensionStore.getExtensions(...)` or `ClassMetadataStore.componentType( ..)`?

I actually see how in the tests you have ...

```javascript
var ClassMetadataStore = require('../dist/ClassMetadataStore').instance;
```

And then make the usage look as though it's a `static` e.g. `ClassMetadataStore.componentType(...)`.